### ISSUE TITLE
Contest id

### ISSUE THREAD

**Issue by CherryGS** [IEEE-NITK/CodePal#84]
Nice extension.
and expect it could provide more variable settings, such as creating folder in the  subdirectory and naming to the contest id, which i'm used to:)

**Comment by adityachirania** [comment]
Yeah that sounds useful! Will try to include this in our next release
